src/prop-types.js

```diff
@@ -373,7 +373,19 @@ export const propTypes = {
     sorting: PropTypes.bool,
     keepSortDirectionOnColumnSwitch: PropTypes.bool,
     toolbar: PropTypes.bool,
-    thirdSortClick: PropTypes.bool
+    thirdSortClick: PropTypes.bool,
```

Comment:
This will throw an error to console, but proceed anyway. Should I write:
```
if (numberOfPagesAround < 1) {
   numberOfPagesAround= 1;
} else if (numberOfPagesAround >= 10) {
   numberOfPagesAround= 10;
}
```
to the src/components/MTableSteppedPaginationInner/index.js to the function renderPagesButton?

Reply:
Oh it must be without the []. So just one of(1,..

Reply:
"Invalid number of arguments, expected 1"
It is the correct way to write PropTypes, but it's just validation. It won't change the value, nor stop the execution.

Reply:
Of course it's just validation. But nevertheless it should not throw an error

Reply:
Warning: Failed prop type: Invalid prop `options.numberOfPagesAround` of value `11` supplied to `MaterialTable`, expected one of [1,2,3,4,5,6,7,8,9,10].

Reply:
Ok but this is a correct warning since it should be below 11 right ?

Reply:
I mean... yes. But now you can put there 100 and it will give you warning, but proceed and try to show you 100 pages (if you have enough data). On the other hand, if you put there -1, it will give you warning but proceed to show you... nothing, no pages.
I wanted to force it to operate in given range.

Reply:
Oh I see. But that's at a different place. So you can min and max the numbers around: max(1, min(10, numerbsAroind))